components/match2/wikis/tft/match_group_input_custom.lua

```diff
@@ -0,0 +1,185 @@
+---
+-- @Liquipedia
+-- wiki=tft
+-- page=Module:MatchGroup/Input/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local FnUtil = require('Module:FnUtil')
+local Lua = require('Module:Lua')
+local Operator = require('Module:Operator')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local MatchGroupInputUtil = Lua.import('Module:MatchGroup/Input/Util')
+local Streams = Lua.import('Module:Links/Stream')
+
+local DEFAULT_BESTOF = 3
+local DEFAULT_MODE = 'team'
+local ALLOWED_VETOES = {'decider', 'pick', 'ban', 'defaultban', 'protect'}
```

Comment:
Is protect actually used or a copy.paste from anotehr wiki?

Reply:
don't see any usage of protect fwiw

Reply:
Protect can be kick this is very likely a leftover from the osu wiki